packages/jest-reporters/package.json

```diff
@@ -5,7 +5,9 @@
   "main": "build/index.js",
   "types": "build/index.d.ts",
   "dependencies": {
+    "@bcoe/v8-coverage": "^0.2.3",
```

Comment:
Perhaps we could move this to "Istanbul/merge-v8-coverage" or "merge-v8-coverage", @demurgos do you have a preference? Would you like to work on this library within the Istanbul organization?

Reply:
Hi,
As discussed in the `v8-coverage` repo, I am working on a 1.0.0 release of the lib. I agree that it would make sense to move it the istanbul org.